lib/runner/extensions/request.command.js

```diff
@@ -35,6 +35,9 @@ var _ = require('lodash'),
         // get a reference to the Auth instance from the original item, so any changes don't have to be synced back
         context.auth = payload.item.getAuth();
 
+        // keep track of no of replays of the request
+        context.replayCount = context.replayCount || 0;
```

Comment:
Yes, if you call it that. But it introduces the concept of "replay" which might not be necessary. Personally, I think it might be good to call it `sendCount` or something, and initialize it to `1` to begin with.